releasenotes/config.yaml

```diff
@@ -49,3 +49,174 @@ sections:
   - [security, Security Issues]
   - [fixes, Bug Fixes]
   - [other, Other Notes]
+template: |
+  ---
+  prelude: >
+      Replace this text with content to appear at the top of the section for this
+      release. All of the prelude content is merged together and then rendered
+      separately from the items listed in other parts of the file, so the text
+      needs to be worded so that both the prelude and the other items make sense
+      when read independently. This may mean repeating some details. Not every
+      release note requires a prelude. Usually only notes describing major
+      features or adding release theme details should have a prelude.
+  features:
+    - |
```

Comment:
Do we know the rendering behavior of a note added directly into `features` instead of a subcategory? I think I remember that being an issue (or behaving strangely at least).

Reply:
If we add something directly to features it gets added to a bulleted list on the parent section before any of the subsections we define below.